docs/slsa.md

```diff
@@ -99,20 +101,27 @@ filesystems, you may not want to include the provenance in these cases.
 
 The provenance format generated by BuildKit is defined by the [SLSA Provenance format](https://slsa.dev/provenance/v0.2).
 
-The following is a selection of fields filled in by BuildKit:
+This section describes how BuildKit populate each field, and whether the
+field gets included when you generate attestations `mode=min` and `mode=max`.
 
 ### `builder.id` [(SLSA)](https://slsa.dev/provenance/v0.2#builder.id)
 
-The `builder.id` field is set to the URL of the build, if available. This is value can be set with `builder-id` attestation parameter.
+Included with `mode=min`.
+
+The `builder.id` field is set to the URL of the build, if available.
 
 ```
     "builder": {
       "id": "https://github.com/docker/buildx/actions/runs/3709599520"
     },
 ```
 
+This value can be set using the `builder-id` attestation parameter.
+
 ### `buildType` [(SLSA)](https://slsa.dev/provenance/v0.2#buildType)
 
+Included with `mode=min`.
```

Comment:
This is also included with `mode=max`, I think we should clarify and mention it's available with both options if possible :tada: